torch/testing/_internal/print_test_stats.py

```diff
@@ -13,7 +14,7 @@
 from pathlib import Path
 from typing import (Any, DefaultDict, Dict, Iterable, Iterator, List, Optional,
                     Tuple)
-from xml.dom import minidom  # type: ignore[import]
```

Comment:
Why did you remove this comment? I thought it was necessary for `mypy` to pass?

Reply:
I don't think it is?

Reply:
Ok cool 😃 that's nice if it's not required

Reply:
never mind...lol weird because locally it recommended I delete the comment:
![image](https://user-images.githubusercontent.com/31798555/109227282-110c1d00-778e-11eb-888a-048f314cf067.png)

Reply:
😮 hmm that's... not good haha, I think I've run into a similar issue before but I forget exactly how to deal with it